cpp/arcticdb/util/test/test_storage_lock.cpp

```diff
@@ -17,6 +17,7 @@ using namespace folly;
 
 TEST(StorageLock, SingleThreaded) {
     SKIP_WIN("StorageLock is not supported");
```

Comment:
The `StorageLock` is actually used in the production code path, so these tests shouldn't have been skipped in Windows in the first place.

They were missed in the original PR, but we must get these tests working on all new platforms.

Reply:
they are segfaulting for mac so I assumed incompatibility